src/DataFrames.jl

```diff
@@ -16,6 +16,8 @@ using Reexport
 using GZip
 using SortingAlgorithms
 
+using FileIO  # remove after read_rda deprecation period
```

Comment:
There's a discussion in #1031 why it should be `FileIO.load()` (because it accepts string names, whereas `RData.load() method has different signature and is not to be called directly, it gets called by `FileIO.load()`). `FileIO` knows about `.rda` format and knows that its support is provided by `RData` package. So if no `RData` package is installed, the user would be asked to install it. By requiring `FileIO` we avoid circular references DataFrames -> RData -> DataFrames.